metricshub-it-common/src/it/java/org/metricshub/it/job/AbstractITJob.java

```diff
@@ -90,6 +90,9 @@ private static void assertMetrics(final Monitor expectedMonitor, final Monitor a
 		for (final Entry<String, AbstractMetric> expectedEntry : expectedMonitor.getMetrics().entrySet()) {
 			final AbstractMetric expectedMetric = expectedEntry.getValue();
 			final String expectedKey = expectedEntry.getKey();
+			if (expectedKey.startsWith("metricshub.job.duration")) {
```

Comment:
We should verify the exact expected metric name, since mismatches could cause side effects later for example, if we introduce a metric like `"metricshub.job.duration.computed"`.